### PR TITLE
README: update to make juttle-engine more prominent

### PR DESCRIPTION
**Pull request by demmer**
Since the more complete installation of juttle will want to use
juttle-engine and not just the CLI, change the introduction and
installation instructions to refer to both options.

### CODE REVIEW

**Comment by demmer** [comment]
@dmajda @dmehra @mccanne What do you guys think about this change as an overall direction. I know it seems a bit strange to refer to juttle-engine here but it is the path we expect many users to go down so it seems appropriate to me.

File: README.md [hunk 1]
```diff
@@ -30,7 +31,13 @@ To use Juttle as a command-line tool, the simplest approach is to install the ju
 $ npm install -g juttle
 ```
 
-You should now have a `juttle` executable in your path which you can use as follows:
+To use the full Juttle engine assembly, run:
```
Comment (dmajda) [inline README.md]: s/engine/Engine/

File: README.md [hunk 2]
```diff
@@ -48,24 +55,15 @@ This produces:
...
 ```
 
-To use Juttle as a JavaScript library, install the juttle package locally as part of your project:
-
-```
-$ npm install juttle
-```
-
-See the [command line reference](./docs/reference/cli.md) for more information about how to configure use the Juttle CLI and how to configure.
+See the [command line reference](./docs/reference/cli.md) for more information about how to configure and use the Juttle CLI. See the [juttle engine README] for command line options and configuration instructions.
```
Comment (dmajda) [inline README.md]: [juttle engine README] — replace by real link?

**Comment by dmajda** [comment]
Referring to Juttle Engine makes total sense to me.

**Comment by dmehra** [comment]
+1 on making Juttle Engine prominent. When looking at the whole README, i noticed that we already had a section at the end "Visualizations with Juttle Engine", that has more detail. Let's link from the new upfront intro of Juttle Engine to that section for more info, so it's not kicking it there disconnected.

File: README.md [hunk 3]
```diff
@@ -30,7 +31,13 @@ To use Juttle as a command-line tool, the simplest approach is to install the ju
 $ npm install -g juttle
 ```
 
-You should now have a `juttle` executable in your path which you can use as follows:
+To use the full Juttle Engine assembly, run:
```
Comment (demmer) [inline README.md]: I just tried and it actually (now) works fine.

**Comment by demmer** [comment]
@dmajda @dmehra @mccanne I updated again, incorporating feedback from the first round. Please take another look.

**Comment by dmajda** [comment]
+1

**Pull request merged**